### PR TITLE
Added an ability to use styles defined in App.xaml within Xamarin.Forms.Player

### PR DESCRIPTION
**Pull request by Shaddix**
Also updated Xam.Plugins.Settings to version 2 (otherwise there are build errors, because 2.0 isn't compatible with 1.5). 
Changed nuspec file to ease the installation (Microsoft.AspNet.SignalR.Client is not installable to the default Xamarin.Forms templates projects). Added an instruction to readme on how to fully utilize the new configuration

### CODE REVIEW

**Comment by kzu** [comment]
Could you squash all commits in a single one for easier review/tracking?

Thanks!

**Comment by Shaddix** [comment]
Done, please check if everything's ok, that's the first time I'm using squash :)

File: src/FormsPlayer/App.cs [hunk 1]
```diff
@@ -13,163 +13,38 @@ namespace Xamarin.Forms.Player
 	/// </summary>
 	public class App : Application
 	{
-		static readonly string FormsAssemblyName = typeof (ContentPage).AssemblyQualifiedName.Substring(typeof (ContentPage).AssemblyQualifiedName.IndexOf(',') + 1).Trim();
+	    private AppController _controller;
```
Comment (kzu) [inline src/FormsPlayer/App.cs]: plz remove `private`, it's redundant, and don't use underscores for field names

File: src/FormsPlayer/App.cs [hunk 2]
```diff
@@ -13,163 +13,38 @@ namespace Xamarin.Forms.Player
...
 
-		AppViewModel viewModel;
-
-		/// <summary>
+	    /// <summary>
```
Comment (kzu) [inline src/FormsPlayer/App.cs:18]: wrong spacing is causing a lot of unnecessary changes :(

**Comment by kzu** [comment]
Other than the minor thing about `private ... _`, it looks great @Shaddix !

**Comment by Shaddix** [comment]
fixed
